src/main/resources/definitions/prisons/orphanage/dpd001-hospital-movements.json

```diff
@@ -0,0 +1,392 @@
+      "schema": {
+        "field": [
+          {
+            "name": "prison_number",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "name",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "date",
+            "type": "date",
+            "display": ""
+          },
+          {
+            "name": "type",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "origin",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "origin_code",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "destination",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "destination_code",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "direction",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "reason",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "escort",
+            "type": "string",
+            "display": ""
+          },
+          {
+            "name": "comment",
+            "type": "string",
+            "display": ""
+          }
+        ]
+      }
+    }
+  ],
+  "policy": [
+    {
+      "id": "caseload",
+      "type": "row-level",
+      "action": ["(origin_code='${caseload}' AND lower(direction)='out') OR (destination_code='${caseload}' AND lower(direction)='in')"],
+      "rule": [
+        {
+          "effect": "permit",
+          "condition": [
+            {
+              "exists": ["${caseload}"]
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "report": [
+    {
+      "id": "hospital-movement-month-mov0027",
+      "name": "Hospital Movements (Last month)",
+      "description": "Prisoner Movements to/from Hospital in the last month",
+      "classification": "list",
+      "version": "v1.0",
+      "render": "HTML",
+      "dataset": "$ref:hospital-dataset",
+      "specification": {
+        "template": "list",
+        "field": [
+          {
+            "name": "$ref:direction",
+            "display": "Direction",
+            "formula": "",
+            "visible": true,
+            "sortable": true,
+            "defaultsort": false,
+            "filter": {
+              "type": "Radio",
+              "staticoptions": [
+                {
+                  "name": "In",
+                  "display": "IN"
```

Comment:
This seems like an odd combination - the `name` which is used by the back-end is in ProperCase, but the value displayed to the user is in uppercase.

I'd would have expected the `display` value to be closer to normal casing, and the `name` value to be lowercase (not that it matters much to the app).